lib/index.js

```diff
@@ -14,13 +14,23 @@ var mapper = require('./mapper');
 
 var Appboy = module.exports = integration('Appboy')
   .channels(['server', 'client'])
-  .endpoint('https://api.appboy.com/users/track')
+  .ensure('settings.datacenter')
   .ensure('settings.appGroupId')
   .ensure('message.userId')
   .mapper(mapper)
   .retries(2);
 
 /**
+ * Initialize.
+ *
+ * @api private
+ */
+
+Appboy.prototype.initialize = function(){
+  this.endpoint = this.settings.datacenter === 'us' ? 'https://api.appboy.com/users/track' : 'https://rest.api.appboy.eu/users/track';
```

Comment:
I see that a datacenter option is required, in which case the logic need not be reversed, but does need a script to be run to set it for legacy users.